pipestat_example/pipeline/pipeline_interface.yaml

```diff
@@ -0,0 +1,6 @@
+pipeline_name: count_lines
+pipeline_type: sample
+var_templates:
+  pipeline: '{looper.piface_dir}/count_lines.py'
+command_template: >
+  python {pipeline.var_templates.pipeline} {sample.file} {sample.sample_name} {pipestat.schema_path} {pipestat.results_file} {pipestat.pipeline_type}
```

Comment:
The pipeline should know what it's `type` is; there is no need to pass this in a CLI arg, through the piface.